### ISSUE TITLE
Action is broken

### ISSUE THREAD

**Issue by meowmeowxw** [er28-0652/setup-ghidra#6]
Hi I'm using the action to test ghidra2dwarf, but it's currently broken.
Example: https://github.com/cesena/ghidra2dwarf/pull/6/checks?check_run_id=1433986748.

Error:

```
Error: Unable to process command '::set-env name=GHIDRA_INSTALL_DIR,::/opt/hostedtoolcache/ghidra/9.1.2/x64' successfully.
Error: The `set-env` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

**Comment by beardypig** [comment]
Same here, I believe the auto-generated PR #5 would actually fix it.

**Comment by meowmeowxw** [comment]
I tried forking and manually merge the PR #5 https://github.com/meowmeowxw/setup-ghidra-new, but it's still not working https://github.com/cesena/ghidra2dwarf/runs/1527645061

**Comment by er28-0652** [comment]
thank you for notification, I'm digging into this.

**Comment by er28-0652** [comment]
@meowmeowxw It finally works. I updated setup-ghidra, did clone your project and confirmed it passed the test. https://github.com/er28-0652/ghidra2dwarf/actions/runs/412780309
As @beardypig  mentioned, I merged PR #5, and also I should have compiled TypeScript source code with updated packages by myself to fully affect the update.

May I ask you to run the action again and confirm if it works?

**Comment by meowmeowxw** [comment]
It seems to work, thanks! :+1:
